utils/docker-ce/Makefile

```diff
@@ -113,6 +113,10 @@ define Package/docker-ce/install
 
 	$(INSTALL_DIR) $(1)/etc/docker
 	$(INSTALL_CONF) ./files/daemon.json $(1)/etc/docker/
+
+	$(INSTALL_DIR) $(1)/etc/sysctl.d
+	$(INSTALL_DATA) ./files/etc/sysctl.d/sysctl-br-netfilter-ip.conf \
+		$(1)/etc/sysctl.d/12-br-netfilter-ip.conf
```

Comment:
Maybe want a comment saying that we chose 12 because we need to override the defaults of `11-br-netfilter.conf` from `kmod-br-netfilter` but I wouldn't let it hold up this PR. Thanks for getting this one!

Reply:
Thanks for the hint